aws-s3-transfer-manager/src/runtime/scheduler.rs

```diff
@@ -58,7 +59,18 @@ impl Scheduler {
     }
 }
 
-pub(crate) type PayloadEstimate = u64;
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Missing Docs. 

Also can probably be `Copy`

Reply:
Thanks, updated with docs and Copy.